### PR TITLE
[PL] added polish HassRespond intent

### PR DESCRIPTION
**Pull request by witold-gren**
## Summary by CodeRabbit

- **New Features**
	- Introduced Polish language responses for the Home Assistant system, enhancing multilingual support.
	- Added intents for greeting, listening status, data inquiries, command capabilities, and creator information.

- **Tests**
	- Implemented test cases for the new Polish responses to ensure proper functionality and accuracy.

### CODE REVIEW

File: tests/pl/homeassistant_HassRespond.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      - "co potrafisz"
+    intent:
+      name: HassRespond
+    response: "Aby dowiedzieć się, o co możesz zapytać, odwiedź: ha. a. kropka. aj. o. slesz. wojs"
+
+  - sentences:
+      - "kto cię stworzył"
+      - "przez kogo zostałeś stworzony"
+      - "kim jest twój twórca"
+      # - "kto jest twoim twórcą"
```
Comment (Uriziel01) [inline tests/pl/homeassistant_HassRespond.yaml:42]: Can we remove the comment?

File: tests/pl/homeassistant_HassRespond.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      name: HassRespond
+    response: "Twoje dane są przesyłane do Twojego serwera Home Assistant."
+
+  - sentences:
+      - "co mogę powiedzieć"
+      - "o co mogę zapytać"
+      - "co potrafisz"
+    intent:
+      name: HassRespond
+    response: "Aby dowiedzieć się, o co możesz zapytać, odwiedź: ha. a. kropka. aj. o. slesz. wojs"
```
Comment (Uriziel01) [inline tests/pl/homeassistant_HassRespond.yaml:36]: Wow, isn't there any other way around it than this kind of notation?

File: sentences/pl/homeassistant_HassRespond.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+language: pl
+intents:
+  HassRespond:
+    data:
+      - sentences:
+          - "(witaj|cześć)[ home assistant]"
```
Comment (Uriziel01) [inline sentences/pl/homeassistant_HassRespond.yaml:6]: Maybe also add `hej`? It's used in wake words so should be quite common for people to try to use it?

File: tests/pl/homeassistant_HassRespond.yaml (hunk 2 above)
Comment (Uriziel01) [reply tests/pl/homeassistant_HassRespond.yaml:36]: This also totally breaks the `Text-to-speech Home Assistant Cloud` as it tries to read it in completely different way.

File: sentences/pl/homeassistant_HassRespond.yaml (hunk 3 above)
Comment (witold-gren) [reply sentences/pl/homeassistant_HassRespond.yaml:6]: added

File: tests/pl/homeassistant_HassRespond.yaml (hunk 1 above)
Comment (witold-gren) [reply tests/pl/homeassistant_HassRespond.yaml:42]: removed

File: tests/pl/homeassistant_HassRespond.yaml (hunk 2 above)
Comment (witold-gren) [reply tests/pl/homeassistant_HassRespond.yaml:36]: I use different text: Aby dowiedzieć się, o co możesz zapytać, odwiedź: home, myślnik, assistant, kropka, i, o, slesz, voice. Sounds normal and domain is correct. There is information on discord that the recommended domain is incorrect.

**Pull request merged**